src/intersection-observer.js

```diff
@@ -103,11 +104,18 @@ export class IntersectionObserver extends Observable {
     this.is3p_ = opt_is3p || false;
     /** @private {boolean} */
     this.shouldSendIntersectionChanges_ = false;
-    /** @private {Array<function>} */
-    this.unlisteners_ = [];
```

Comment:
This code was never used. If we want to kill entire ad tags, we'd need to add this again.